includes/find3server.service

```diff
@@ -10,7 +10,7 @@ RuntimeDirectory=find3server
 RuntimeDirectoryMode=0775
 User=find3
 Group=openhabian
-ExecStart=/usr/sbin/find3server -mqtt-server %MQTT_SERVER -mqtt-admin %MQTT_ADMIN -mqtt-pass %MQTT_PASS -port %FIND3_PORT
+ExecStart=/usr/sbin/find3server -port %FIND3_PORT
```

Comment:
Are you sure it'll take the remaining parameters off /etc/default/find3server ? Why ? And why don't you put the port number there, too ?

Reply:
It does, I tested. I looked at the source code and it does not support setting the port from the env.

Reply:
I'm confused. Are you saying it takes all those MQTT_* env variables but not the port ?

[18:49:17] root@openhab:/opt/openhabian#  cat /etc/default/find3server
# FIND server service configuration
# see https://www.internalpositioning.com/server/
#
MQTT_SERVER=openhabianpi:1883
MQTT_ADMIN=openhab2
MQTT_PASS=

Reply:
Yes, that is what I am saying.

Reply:
well ok then
can we use $(grep PORT /etc/default/find3server) or similar in .service ?
if no I'm fine with this, too.

Reply:
No we can't, that is a bash substitution which won't work in .service files. I am not aware of any way that would work in .service files.